src/Illuminate/Console/Scheduling/ManagesFrequencies.php

```diff
@@ -61,6 +65,18 @@ private function inTimeInterval($startTime, $endTime)
         };
     }
 
+    /**
+     * Check if endTime is considered greater for humans.
+     *
+     * @param string $startTime
+     * @param string $endTime
+     * @return bool
+     */
+    private function endTimeGreater($startTime, $endTime)
+    {
+        return explode(':', $startTime)[0] > explode(':', $endTime)[0];
```

Comment:
You are only checking the hours. What if `endTimeGreater('23:50', '23:00')`

Reply:
@jn-jairo and tbh, that double explode() is probably not the ideal solution.. :)

Reply:
Thanks for your insights, I will definitely add comparison for minutes. I'll look for an alternative way rather than the explode.

Reply:
@MattMangoni  I felt the same. I'm actually rethinking the naming. I didn't wanted the function name to be too long. But I'll give a second try.